static/js/panel.js

```diff
@@ -0,0 +1,69 @@
+var panel = (function () {
```

Comment:
@brockwhittaker maybe we should rename this module to `desktop_notifications_panel.js`?  I feel like this name is pretty under-specified within the broader codebase.